objects/src/testing/account_code.rs

```diff
@@ -160,8 +160,13 @@ impl AccountCode {
         code
     }
 
+    /// Creates a mock [AccountCode] with specific code and assembler
+    pub fn mock_specific(source_code: &str, assembler: Assembler) -> AccountCode {
```

Comment:
Naming could potentially be improved here.

Reply:
I would probably name this `mock_with_code()`.

Also @igamigo - do we not already have something like this?